new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporter.java

```diff
@@ -247,37 +260,35 @@ public static void initAll(@Namespaces(6) List<String> namespaces) {
     assertNotNull(namespaces.get(5), "Namespace list is null");
     final String nginxNamespace = namespaces.get(5);
 
+    logger.info("Get a unique namespace for domain3");
+    assertNotNull(namespaces.get(6), "Namespace list is null");
+    domain3Namespace = namespaces.get(6);
+
+    logger.info("Get a unique namespace for domain3");
```

Comment:
cun-n-paste error " logger.info("Get a unique namespace for domain3"); '  is repeated twice

Reply:
corrected